content/contributing/documentation/content_guidelines.rst

```diff
@@ -154,6 +155,28 @@ are also displayed in a dynamic *navigation bar* on the side.
    - :ref:`RST cheat sheet: headings <contributing/headings>`
    - :ref:`RST cheat sheet: specialized directives <contributing/specialized-directives>`
 
+.. _contributing/organizing-documentation:
+
+Organizing the documentation
+============================
+
+When writing documentation about a given topic, try to keep related pages organized.
```

Comment:
I knew what you meant with "related pages" because I knew the topic already. It could be interpreted differently, such as "related topics," rather than the files that are in the same folder.

Reply:
I inspired myself from the note on the top of the page:
> the other documents related to this section of the documentation

Do you want to change this formulation too?

Reply:
That's something you modified in this PR ^^ It wasn't written like that before.
The meaning is different though, to me.

"the other documents related to this section of the documentation."
--> "related" works here since the topics are related in meaning.

"When writing documentation about a given topic, try to keep related pages organized."
--> I want to understand "related" from a "topic" understanding, especially since it's written a few words before. 

I think it's all good now :)